docs/chaos-engineering/chaos-faults/azure/azure-web-app-access-restrict.md

```diff
@@ -0,0 +1,287 @@
+- Ensure that you have sufficient Azure access to detach and attach a disk. 
+- We will use azure [ file-based authentication ](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorization#use-file-based-authentication) to connect with the instance using azure GO SDK in the experiment. For generating auth file run `az ad sp create-for-rbac --sdk-auth > azure.auth` Azure CLI command.
+- Ensure to create a Kubernetes secret having the auth file created in the step in `CHAOS_NAMESPACE`. A sample secret file looks like:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-secret
+type: Opaque
+stringData:
+  azure.auth: |-
+    {
+      "clientId": "XXXXXXXXX",
+      "clientSecret": "XXXXXXXXX",
+      "subscriptionId": "XXXXXXXXX",
+      "tenantId": "XXXXXXXXX",
+      "activeDirectoryEndpointUrl": "XXXXXXXXX",
+      "resourceManagerEndpointUrl": "XXXXXXXXX",
+      "activeDirectoryGraphResourceId": "XXXXXXXXX",
+      "sqlManagementEndpointUrl": "XXXXXXXXX",
+      "galleryEndpointUrl": "XXXXXXXXX",
+      "managementEndpointUrl": "XXXXXXXXX"
+    }
+```
+- If you change the secret key name (from `azure.auth`) please also update the `AZURE_AUTH_LOCATION` ENV value on `experiment.yaml`with the same name.
+:::
+
+## Default Validations
+:::info
+- Azure Disk should be connected to an instance.
+:::
+
+## Experiment tunables
+<details>
+    <summary>Check the Experiment Tunables</summary>
+    <h2>Mandatory Fields</h2>
+    <table>
+        <tr>
+            <th> Variables </th>
+            <th> Description </th>
+            <th> Notes </th>
+        </tr>
+        <tr> 
+            <td> AZURE_WEB_APP_NAMES </td>
+            <td> Name of azure web app services to target.</td>
+            <td> Provide comma separated names web app stop chaos</td>
+        </tr>
+        <tr>
+            <td> RESOURCE_GROUP </td>
+            <td> The resource group of the target web app</td>
+            <td> </td>
+        </tr> 
+    </table>
+    <h2>Optional Fields</h2>
+    <table>
+        <tr>
+            <th> Variables </th>
+            <th> Description </th>
+            <th> Notes </th>
+        </tr>
+        <tr>
+            <td> RULE_NAME </td>
+            <td> Provide the rule name that should be added as part of chaos injection</td>
+            <td> If not provided it will use a default name <code>litmus-experiment-rule</code></td>
+        </tr>
+        <tr>
+            <td> IP_ADDRESS_BLOCK </td>
+            <td> Provide the IP address/CIRD range for the rule</td>
+            <td>  Default is <code>0.0.0.0/0</code></td>
+        </tr>
+        <tr>
+            <td> ACTION </td>
+            <td> Provide the action you want to perfrom with the rule</td>
+            <td> Accepts "allow"/"deny". Default is "deny"</td>
+        </tr>
+        <tr>
+            <td> PRIORITY </td>
+            <td> Provide the priority of the rule</td>
+            <td>  Default is "300"</td>
```

Comment:
would you like to add more info here? i.e, scale

Reply:
We can at least mention whether a lower number is more or less in priority.